hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/resolver/backend/HttpBackendResolver.java

```diff
@@ -40,41 +41,41 @@ public HttpBackendResolver(FQNResolver fqnResolver) {
     Protocol protocol = EnrichedSpanUtils.getProtocol(event);
 
     if (protocol == Protocol.PROTOCOL_HTTP || protocol == Protocol.PROTOCOL_HTTPS) {
-      String backendUriStr = null;
+      String urlStr = null;
       String path = null;
       if (SpanAttributeUtils.containsAttributeKey(event, HTTP_URL_ATTR)) {
```

Comment:
I think we should also re factor this to use first class field now, since we are already touching the code. Like, we can get the `host`, `port` and `path` directly from the event as first class field

(_I am not sure if the host and port are available as first class fields. If not, we can use the URL to derive that_)

Cc @buchi-busireddy @tim-mwangi @avinashkolluru

Reply:
Yes, we have the host(which can potentially contain both host and port fields and is handled internally by `getBackendEntityBuilder` method) and path as first class fields. Made this change